pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -408,6 +423,15 @@ func (r *ReconcileConnection) Reconcile(ctx context.Context, request reconcile.R
 		// License becomes available. Therefore, if we fail to query the Tier, we exclude NetworkPolicy from reconciliation
 		// and tolerate errors arising from the Tier not being created.
 		includeEgressNetworkPolicy = tierAvailable && licenseActive
+
+		clusterInformation, err := utils.FetchClusterInformation(ctx, r.cli)
+		if err != nil {
+			r.status.SetDegraded(operatorv1.ResourceReadError, "Error querying clusterInformation", err, reqLogger)
+			return reconcile.Result{}, err
+		}
+		if len(clusterInformation.Spec.CNXVersion) != 0 {
```

Comment:
nit, but I don't think we need this branch check - if the length is 0, then we'll get `cnxVersion = ""` which is the same as if we did nothing.

Reply:
What about OSS clusters - they don't have this field. Should we use the OSS version for them? Or should we disable the ClusterInformation watch for OSS clusters if we're not using it?